### DISCUSSION TITLE
Getting a 403 error code when publishing a notice

### DISCUSSION THREAD

**Discussion by teslavitas** [OP-TED/eForms-SDK#810] [Q&A] [answered]
I am getting a 403 Forbidden response when submitting a notice.
It is the first time I am trying to publish a notice in production.
On the preview environment everything works.
A also get a 403 error when search for a notice on production with my api key using the swagger page https://enotices2.ted.europa.eu/esenders/webjars/swagger-ui/index.html#/Notice/searchProceduresEsentool .
I generated a new token, but it didn't help.
Could you check it?
My e-mail is eSenderProd at cdm.dk
requests were sent around 14:00 UTC 26.01.2024

**Comment by teslavitas** [comment, marked as answer]
It started to work after doing this:
_If you use the the eNotices2 API, after you generate your key, please log into the [eNotices2 website](https://enotices2.ted.europa.eu/) at least once to pair your account._

**Comment by bertrand-lorentz** [comment]
Thnks for posting the solution to your problem.

In the future, for any problems with the eNotices2 application or its API, please contact the helpdesk at [ted@publications.europa.eu](mailto:ted@publications.europa.eu)
